packages/botonic-react/src/components/message.jsx

```diff
@@ -13,6 +13,12 @@ const DefaultMessage = styled.div`
   font-family: Arial, Helvetica, sans-serif;
   border-radius: 8px;
 `
+const TextMessage = styled.div`
```

Comment:
Maybe I would change the name, it's not mandatory that this is gonna have a Text in it.